src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx

```diff
@@ -283,9 +283,11 @@ The following List Price tables identify per-unit prices. For more information,
         </tbody>
       </table>
 
-#### Add-ons 
+The List Price table below includes pricing details for several add-ons, including: 
 
-Here are pricing details for exceeding default [data retention](/docs/data-apis/manage-data/manage-data-retention) and [included synthetic checks](/docs/synthetics/synthetic-monitoring/using-monitors/monitor-limits). 
+* Exceeding default [data retention](/docs/data-apis/manage-data/manage-data-retention)
```

Comment:
@zuluecho9 I'm not quite sure why you're including a bullet list of items that seem to be covered in the table itself.

Reply:
just because the table I find a bit confusing and want to make it clear what the items relate to